client/state/data-layer/wpcom/sites/media/index.js

```diff
@@ -27,23 +30,34 @@ const log = debug( 'calypso:middleware-media' );
  * @param  {Object}  action Action object
  * @return {Promise}        Promise
  */
-export function requestMedia( { dispatch, getState }, { siteId, query } ) {
-	if ( isRequestingMedia( getState(), siteId, query ) ) {
+export function requestMedia( { dispatch, getState }, action ) {
+	if ( isRequestingMedia( getState(), action.siteId, action.query ) ) {
```

Comment:
Oh I missed this, but we can actually remove this check/early return. The wpcom-http layer handles deuduping the same http GET requests

Note that since we don't need to know isRequesting state, we can also remove this selector, and the related  _SUCCESS/_ERROR/_REQUESTING actions in another PR.

Reply:
I removed this in https://github.com/Automattic/wp-calypso/pull/18022/commits/2b55d9f1f2ff3dbc7431a2f020f9ff18f8edf2b6 . Thank you!